buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -124,7 +124,7 @@
       "allowOverride": true
     },
     "BuildArguments": {
-      "value": "--skip-prereqs --configuration $(BuildConfiguration)  --docker centos.7 --targets Default",
+      "value": "--skip-prereqs --configuration $(BuildConfiguration)  --docker centos.7 --targets Default --env-vars 'NUGET_API_KEY=$(NUGET_API_KEY)','GITHUB_PASSWORD=$(GITHUB_PASSWORD)','REPO_PASS=$(REPO_PASS)'",
```

Comment:
Reposting comment that GitHub marked out of date:

Why not `CONNECTION_STRING`? Doesn't that need to be secret, too?

Reply:
I based the variables I included off of what the pipebuild complained was missing - I didn't see anything about CONNECTION_STRING in https://devdiv.visualstudio.com/DevDiv/Net%20CoreFX/_build/index?buildId=1461297&_a=summary, so I didn't include it. I guess maybe it isn't used?

Reply:
It looks like that build may have been affected by https://github.com/dotnet/core-eng/issues/1906#issuecomment-371960113, so pipebuild just missed it and passed it in as plaintext. To be sure, I think it's worth running another build now to get a fresh list. It's definitely used, though, when publish steps use the publisher tool: https://github.com/dotnet/core-setup/blob/4f2e209bc07a80403daf08381dfaafcc74a2c419/build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs#L28